integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/receipt/strategy/DpoConversationStrategy.java

```diff
@@ -5,7 +5,7 @@
 import org.springframework.stereotype.Component;
 
 @Component
-public class EduConversationStrategy implements ConversationStrategy {
+public class DpoConversationStrategy implements ConversationStrategy {
 
     @Override
     public void checkStatus(Conversation conversation) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1186)